File: Baekjoon/Class1/33.2920.js
const inputFile = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = require("fs").readFileSync(inputFile, "utf8").trim();

let cursor = 0;
const notes = input.split(" ").map(Number);
let isAscending = true, isDescending = true;
let prev = notes[cursor++];
for(; cursor < notes.length; cursor++) {
    const cur = notes[cursor];
    if(prev >= cur) isAscending = false;
    if(prev <= cur) isDescending = false;
    prev = cur;
}
if(isAscending) console.log("ascending");
else if(isDescending) console.log("descending");
else console.log("mixed");

File: Baekjoon/Class1/33.2920.java
import java.io.*;
import java.util.StringTokenizer;

class Main {
    public static void main(String[] args) throws IOException {
        boolean isAscending = true, isDescending = true;
        int prev, cur;
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        prev = Integer.parseInt(st.nextToken());
        while(st.hasMoreTokens()) {
            cur = Integer.parseInt(st.nextToken());
            if (prev >= cur) isAscending = false;
            if (prev <= cur) isDescending = false;
            prev = cur;
        }
        if(isAscending) System.out.print("ascending");
        else if(isDescending) System.out.print("descending");
        else System.out.print("mixed");
        br.close();
    }
}